src/main/webapp/style.css

```diff
@@ -5,3 +5,32 @@ h3 {
 .srp-container {
   display: flex;
 }
+
+.header-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between; 
+  margin-left: 50%;
```

Comment:
I couldn't figure out a different way to get the logo naming to be in the center, but this also doesn't seem like the most ideal way...